tag-historian/nitaghistorian.yml

```diff
@@ -609,6 +609,39 @@ definitions:
         description: The workspace ID for the tag. If this is not specified, the default workspace will be used.
         type: string
         example: 0c80cf49-54e9-4e92-b117-3bfa574caa84
+  UpdateMetadata:
+    title: Update Metadata
+    description: Defines metadata values to update on a tag.
```

Comment:
Should we document that this route can be used for creating the initial tag metadata but also for updating?